### PR TITLE
Added the ability to specify the channels Ids

### PR DESCRIPTION
**Pull request by abdellahaski**

### CODE REVIEW

File: config.yml-sample [hunk 1]
```diff
@@ -13,3 +13,13 @@ input_channel_names:
 output_channel_names: 
   - 'Output Channel Name 1'
   - 'Output Channel Name 2'
+
+# The channel IDs that you'd like to forward messages from. (You can get a channel ID by forwarding any message from it to @userinfobot , and removing the -100 from the start )
+input_channel_ids:
+  - 'Input Channel ID 1' # without quotes ''
```
Comment (voidbar) [inline config.yml-sample]: remove quotes from the examples

File: config.yml-sample [hunk 2]
```diff
@@ -13,3 +13,13 @@ input_channel_names:
 output_channel_names: 
   - 'Output Channel Name 1'
   - 'Output Channel Name 2'
+
+# The channel IDs that you'd like to forward messages from. (You can get a channel ID by forwarding any message from it to @userinfobot , and removing the -100 from the start )
```
Comment (voidbar) [inline config.yml-sample:17]: give example for how to set this list as an empty one. i.e: `input_channel_ids: []`

**Review by voidbar — CHANGES_REQUESTED** [review]
Thanks for the contribution! was thinking about adding it myself! seems ok but I have few small comments

**Comment by abdellahaski** [comment]
@voidbar you're welcome, I just committed the edits that you requested

**Comment by jerung3000** [comment]
Hi voidbar

When I run this repo..Igot error like this..

Traceback (most recent call last):
  File "C:\PYTHON\forwardgram\forwardgram\forwardgram.py", line 49, in <module>
    start(config)
  File "C:\PYTHON\forwardgram\forwardgram\forwardgram.py", line 22, in start
    input_channels_entities.append(InputChannel(d.entity.id, d.entity.access_hash))
AttributeError: 'Chat' object has no attribute 'access_hash'

How solve this error..?Thank you..

**Comment by abdellahaski** [comment]
@jerung3000 Can you open an issue, and include your config file? (don't forget to remove your api_id/hash)

**Pull request merged**